pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -570,7 +571,11 @@ func (plugin *kubenetNetworkPlugin) addContainerToNetwork(config *libcni.Network
 	// The network plugin can take up to 3 seconds to execute,
 	// so yield the lock while it runs.
 	plugin.mu.Unlock()
-	res, err := plugin.cniConfig.AddNetwork(config, rt)
+	// Because the default remote runtime request timeout is 4 min,so set slightly less than 240 seconds
+	// Todo get the timeout from parent ctx
+	cniTimeoutCtx, cancelFunc := context.WithTimeout(context.Background(), 220*time.Second)
```

Comment:
Can 220 be a const somewhere, please

Reply:
@liubin @thockin const yes please :)

Reply:
Just put the const into pkg/kubelet/dockershim/network/network.go and use it from everywhere else that needs it.